spec/memfs/file_spec.rb

```diff
@@ -950,6 +950,30 @@ module MemFs
       end
     end
 
+    describe '.write' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.